src/common/tests/testUtils.js

```diff
@@ -43,8 +50,9 @@ function isRect2DObject(instance) {
       'moveTopLeftTo', 'moveTopRightTo', 'moveBottomRightTo', 'moveBottomLeftTo',
       'moveLeftCenterTo', 'moveTopCenterTo', 'moveRightCenterTo', 'moveBottomCenterTo',
       'normalize', 'setCoords', 'getCoords', 'onEdge',
-      'contains', 'properlyContains', 'properlyContainsCoord', 'intersects',
-      'unite', 'split', 'splitX', 'splitY',
+      'contains', 'containsCoord', 'properlyContains',
+      'properlyContainsCoord', 'intersects', 'intersectsCoord',
+      'unite', 'uniteCoord', 'split', 'splitX', 'splitY',
```

Comment:
Please be noted that unite/split/intersect methods series are not implemented yet. So they're not in doc/spec at present.